packages/server/src/routes/provider/alert/create.svelte

```diff
@@ -546,26 +586,71 @@
         <button
           class="delete"
           aria-label="close"
-          on:click={() => (isIconSelectorOpen = false)} />
+          on:click={() => {
+            isIconSelectorOpen = false;
+            isCustomIconSelectorOpen = false;
+          }} />
       </header>
-      <section class="modal-card-body">
-        <div class="icon-choiced-container">
-          <div class="icon-choices">
-            {#each Object.values(icons) as icon}
-              <div class="icon-container">
-                <button
-                  class="button is-large"
-                  class:has-background-info={tempIcon === icon}
-                  on:click={() => (tempIcon = icon)}>
-                  <span class="icon is-large">
-                    <i class={`${icon} fa-2x`} />
-                  </span>
-                </button>
+      {#if isCustomIconSelectorOpen}
+        <section class="modal-card-body">
+          <div class="icon-choiced-container">
+            {#if iconSearchResults.length}
+              <div class="icon-choices">
```

Comment:
It looks like it'd be ok for the `.icon-choices` container to render whether we have results or not. Since `iconSearchResults` is now always an array, we can remove this conditional and only have one conditional around the "No results found".